src/js/audit-submissions.js

```diff
@@ -1,18 +1,48 @@
 (function () {
   function init() {
+    const START_SUBMISSION_URL = '../audit/new/step-1/';
     const submissions_table = document.getElementById('audit-submissions');
     const uei_link = submissions_table.querySelector('th .usa-link');
     uei_link.setAttribute('href', '#modal-uei-info');
     uei_link.setAttribute('aria-controls', 'modal-uei-info');
     uei_link.setAttribute('data-open-modal', '');
+    const modal_uei_second_button = document.querySelectorAll(
+      '#modal-uei-info .usa-button-group__item'
+    )[1];
+    modal_uei_second_button.remove();
 
-    const button_group = document.querySelector(
-      '#modal-uei-info .usa-button-group'
+    const terms_form = document.querySelector('#start-new-submission');
+    terms_form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      console.log('FORM SUBMITTED');
```

Comment:
Ideally we shouldn't log anything to the console in the success case.

Reply:
TY!